flathub.json

```diff
@@ -1,3 +1,3 @@
 {
-    "only-arches": ["x86_64", "i386"]
+  "skip-arches": ["x86_64", "i386"]
```

Comment:
Wait what?

Reply:
Yeah. I should have marked this PR as WIP. Sorry.

Basically, I just wanted to test if without the `--concatenate` build options on PyQT, the build passed on arm & aarch.

I'll delete the commit and force push later.